dependency-check-maven-config.xml

```diff
@@ -1,244 +1,162 @@
-        <cve>CVE-2014-0050</cve>
+        <sha1>6EECC43A0883EF7C2A8E4BAF3AD45E24D8119ECC</sha1>
+        <cpe>cpe:/a:ffmpeg:ffmpeg:-</cpe>
     </suppress>
 
+    <!-- proxy-camel-servlet false positives - these are all because the proxy-camel-servlet is getting confused with the camel-servlet-->
     <suppress>
-        <notes>
-            CVE-2016-1000031: Applies to commons-fileupload-1.2.1, suppressing because the vulnerable class DiskFileItem is not used in the project
-        </notes>
-        <cve>CVE-2016-1000031</cve>
-    </suppress>
+        <notes>This is an issue with Camel version before 2.16.1
+            OWASP appears to have confused the internal proxy-camel-servlet
+            version with the overall Camel version - marking as false positive.
 
-    <suppress>
-        <notes><![CDATA[
-        false positive the effected camel version is 2.12 this uses a later version and does use the XSLT component
-      file name: proxy-camel-servlet-2.9.0-SNAPSHOT.jar
-      ]]></notes>
+            file name: proxy-camel-servlet-2.11.0-SNAPSHOT.jar
+        </notes>
+        <filePath regex="true">.*proxy-camel-servlet.*.jar</filePath>
+        <cve>CVE-2015-5344</cve>
         <cve>CVE-2014-0002</cve>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-        false positive the effected camel version is 2.12 this uses a later version and does use the XSLT component
-   file name: proxy-camel-servlet-2.9.0-SNAPSHOT.jar
-   ]]></notes>
         <cve>CVE-2014-0003</cve>
+        <cve>CVE-2017-3159</cve>
     </suppress>
 
+    <!-- tika -->
     <suppress>
-        <notes><![CDATA[
-        this CVE is generating a lot of false positives it should only include jackson-dataformat-xml jar
-         but it is catching all jackson dependencies unfortunately I can't fix it so if we start depending
-          on a vulnerable version of jackson-dataformat-xml this will still suppress it.
-   ]]></notes>
-        <cve>CVE-2016-3720</cve>
-    </suppress>
-
-    <suppress>
-        <notes><![CDATA[
-   file name: commons-beanutils-1.8.3.jar
-   shiro-core has a dependency on this but it doesn’t expose commons-beanutils to user input so it wouldn't pose a risk like the struts library that is called out in the CVE
-   ]]></notes>
-        <sha1>686EF3410BCF4AB8CE7FD0B899E832AABA5FACF7</sha1>
-        <cve>CVE-2014-0114</cve>
-    </suppress>
-
-    <suppress>
-        <notes><![CDATA[
-   file name: ffmpeg-3.1.1_1-bin.zip: ffmpeg.exe
-   Reported CVE's are vulnerabilities in earlier versions of FFmpeg
-   ]]></notes>
-        <sha1>6EECC43A0883EF7C2A8E4BAF3AD45E24D8119ECC</sha1>
-        <cpe>cpe:/a:ffmpeg:ffmpeg:-</cpe>
-    </suppress>
-
-    <suppress>
-        <notes><![CDATA[
-        false positive CVE is unrelated
-   file name: platform-filter-delegate-*.jar
-   ]]></notes>
-        <cve>CVE-2005-0861</cve>
+        <notes>
+            This is an issue with the jmatio library included in the tika-parser 1.13. The problem
```

Comment:
Ok. Reading the comment I thought that upgrading to 1.14 or 1.15 would fix the problem. Should we clarify that the false positive exists even with those versions?